### PR TITLE
Habib/career page fix request

### PR DESCRIPTION
**Pull request by habib-deriv**
Changes:

-  Remove New Locations and BeSquare Banner

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/layout/besquare/footer.js [hunk 1]
```diff
@@ -50,84 +50,86 @@ const BeSquareFooter = () => {
             link_to: 'https://twitter.com/besquarebyderiv',
         },
     ]
-
+    const besquare_banner = false
```
Comment (sean-binary) [inline src/components/layout/besquare/footer.js:53]:
```suggestion
    const hasBesquareBanner = false
```

Comment (sean-binary) [reply src/components/layout/besquare/footer.js:53]: Let's rename it this way since it's a boolean.

File: src/components/layout/besquare/nav.js [hunk 2]
```diff
@@ -4,6 +4,8 @@ import { besquare_signup_url } from 'common/constants'
 import { localize } from 'components/localization'
 import BeSquareLogo from 'images/svg/be-square/be-square-logo.svg'
 
+const be_square_apply = false
```
Comment (sean-binary) [inline src/components/layout/besquare/nav.js:7]:
```suggestion
const hasApplyButton = false
```

Comment (sean-binary) [reply src/components/layout/besquare/nav.js:7]: Same here.

**Review by kevinw-binary — APPROVED** [review]

**Pull request merged**